From 17ec1bce74419f8193c0246dbde2d2fdd6b1b4e3 Mon Sep 17 00:00:00 2001
From: Peter Korsgaard <peter@korsgaard.com>
Date: Wed, 12 May 2021 09:18:33 +0200
Subject: [PATCH] Makefile: drop explicit undefine PYLIB for compatibility with
 make 3.81

GNU make 3.81 does not like the explicit tab-indented undefine:

Makefile:41: *** commands commence before first target.  Stop.

Dropping the indentation also doesn't work:

Makefile:41: *** missing separator.  Stop.

So simply drop the undefine logic.  As explained in the GNU make manual,
undefine is not commonly needed as E.G. ifdef checks for a non-zero string:

https://www.gnu.org/software/make/manual/html_node/Undefine-Directive.html
https://www.gnu.org/software/make/manual/html_node/Conditional-Syntax.html

Fixes:
http://autobuild.buildroot.net/results/cf7c4f360f5464c700788cc8299fd086544c80e8/build-end.log

Signed-off-by: Peter Korsgaard <peter@korsgaard.com>
Upstream: https://marc.info/?l=linux-rt-users&m=162080462211139&w=2
[Julien: rebased patch on version 2.7]
Signed-off-by: Julien Olivain <ju.o@free.fr>
---
 Makefile | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/Makefile b/Makefile
index f656e34..1206f40 100644
--- a/Makefile
+++ b/Makefile
@@ -52,11 +52,6 @@ else
 $(warning libcpupower disabled, building without --deepest-idle-state support.)
 endif
 
-# Check for errors, such as python3 not available
-ifeq (${PYLIB},)
-	undefine PYLIB
-endif
-
 MANPAGES = src/cyclictest/cyclictest.8 \
 	   src/pi_tests/pi_stress.8 \
 	   src/ptsematest/ptsematest.8 \
-- 
2.47.0

